### ISSUE TITLE
Conserved Moieties Default Behavior

### ISSUE THREAD

**Issue by 0u812** [sys-bio/roadrunner#211]
I switched the default behavior to have conserved moieties enabled by default a few weeks ago, but I'm not sure it's the best thing to do. I just found out the hard way that you can't have events and conserved moieties enabled at the same time. Maybe I can just disable conserved moieties for models with events in them. Example (please excuse the hacky model):
```
import tellurium as te

m = te.loada('''
x ->  ; k*x
  -> t; 1

k = 0.1
x=10

at t>7: x=10, t=t-24;
''')

r = m.simulate(0,50,50)
print(r)
m.plot()
```

Output:
```
RuntimeError: Cannot perform moeity conversion when floating species are have events. The floating species, t has event _E0.
 To disable conserved moeity conversion, blah blah blah...
```

@hsauro, what do you think?

Btw I'm also going to fix the "are have events" error message.

**Comment by hsauro** [comment]
ok disable it. The problem comes however when someone wants to do steady
state analysis and there are conserved cycles, what do we do in that case?
The docs are so obscure on how to switch on and off the conservation cycle
analysis.  eg searching one finds:

RoadRunner.conservedMoietyAnalysis
<http://sys-bio.github.io/roadrunner/python_docs/api_reference.html?highlight=conservedmoietyanalysis#RoadRunner.RoadRunner.conservedMoietyAnalysis>

Enables / Disables conserved moiety analysis (boolean).

but what does that mean? There is no example.

**Comment by 0u812** [comment]
I know, they're terrible, here's the correct way:
```
Config.setValue(Config.LOADSBMLOPTIONS_CONSERVED_MOIETIES, False)
```

**Comment by hsauro** [comment]
There is also the conservedMoietyAnalysis flag which I can never remember
how to use. I've added an issue to github for Kiri to add some examples at
specific points in the docs. I'll augment the issue with use of C
onfig.setValue(Config.LOADSBMLOPTIONS_CONSERVED_MOIETIES, False) as well

**Comment by hsauro** [comment]
PS Does Config.setValue(Config.LOADSBMLOPTIONS_CONSERVED_MOIETIES, False)
set it so that the option is there from session to session or is it only
for the current session? If not how do we set session to session options?

**Comment by 0u812** [comment]
The Config options are global, so they're in effect from the moment you
```
import roadrunner
```
until the module is unloaded by Python (usually at the end of the script).

**Comment by hsauro** [comment]
Is it possible to set configurations that last session to session other
than editing the conf file?

H

**Comment by 0u812** [comment]
The only way to do it currently is to edit the file...

On October 2, 2015 11:42:26 AM PDT, Herbert Sauro <notifications@github.com> wrote:
>Is it possible to set configurations that last session to session other
>than editing the conf file?
>
>H
>
>On Fri, Oct 2, 2015 at 11:31 AM, Kyle Medley <notifications@github.com>
>wrote:
>
>> Closed #211 <https://github.com/sys-bio/roadrunner/issues/211> via
>4079d95
>>
><https://github.com/sys-bio/roadrunner/commit/4079d95f0358d799e0c816b62862abfe41b8e2bb>
>> .
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/sys-bio/roadrunner/issues/211#event-425571615>.
>>
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/sys-bio/roadrunner/issues/211#issuecomment-145120216

-- 
Sent from my TARDIS with K-9 Mail. Wibbly-wobbly wordy gurdy.

**Comment by hsauro** [comment]
Do you think we users should have someway to programmatically change the
file?

H

**Comment by 0u812** [comment]
I can make a method for that

On October 2, 2015 1:16:41 PM PDT, Herbert Sauro <notifications@github.com> wrote:
>Do you think we users should have someway to programmatically change
>the
>file?
>
>H
>
>On Fri, Oct 2, 2015 at 11:59 AM, Kyle Medley <notifications@github.com>
>wrote:
>
>> The only way to do it currently is to edit the file...
>>
>> On October 2, 2015 11:42:26 AM PDT, Herbert Sauro <
>> notifications@github.com> wrote:
>> >Is it possible to set configurations that last session to session
>other
>> >than editing the conf file?
>> >
>> >H
>> >
>> >On Fri, Oct 2, 2015 at 11:31 AM, Kyle Medley
><notifications@github.com>
>> >wrote:
>> >
>> >> Closed #211 <https://github.com/sys-bio/roadrunner/issues/211> via
>> >4079d95
>> >>
>> ><
>>
>https://github.com/sys-bio/roadrunner/commit/4079d95f0358d799e0c816b62862abfe41b8e2bb
>> >
>> >> .
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >>
><https://github.com/sys-bio/roadrunner/issues/211#event-425571615>.
>> >>
>> >
>> >
>> >---
>> >Reply to this email directly or view it on GitHub:
>>
>>https://github.com/sys-bio/roadrunner/issues/211#issuecomment-145120216
>>
>> --
>> Sent from my TARDIS with K-9 Mail. Wibbly-wobbly wordy gurdy.
>>
>> —
>> Reply to this email directly or view it on GitHub
>>
><https://github.com/sys-bio/roadrunner/issues/211#issuecomment-145126392>.
>>
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/sys-bio/roadrunner/issues/211#issuecomment-145144103

-- 
Sent from my TARDIS with K-9 Mail. Wibbly-wobbly wordy gurdy.

**Comment by hsauro** [comment]
ok, make sure you it on the C api and Python.

H
